plugins/idea/pluginBase/src/main/java/com/imaginea/kodebeagle/base/util/SearchUtils.java

```diff
@@ -143,27 +144,35 @@ private JsonArray getJsonHitsArray(final JsonObject hitsObject) {
         return hitsObject.getAsJsonArray(HITS);
     }
 
-
-    public final String getESResultJson(final String esQueryJson, final String url) {
+    public final String getResponseJson(final String url, final String query, final boolean appendNewLineInReponse) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) The Cyclomatic Complexity of this method "getResponseJson" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)